.github/workflows/build.yml

```diff
@@ -27,6 +27,7 @@ jobs:
           echo "$PROTECTEDACCOUNT_PROPERTIES" > src/test/resources/protectedaccount.properties
           echo "$SERVICE_ACCOUNT_PROPERTIES" > src/test/resources/serviceaccount.properties
           echo "$OAUTHACCOUNT_PROPERTIES" > src/test/resources/oauthaccount.properties
+          echo "$APPLICATION_DEFAULT_PROPERTIES" > src/test/resources/applicationdefault.properties
```

Comment:
Ok I have added this secret (APPLICATION_DEFAULT_PROPERTIES) to the repository that contains the data that @thomasbanghart sent me.